### ISSUE TITLE
mongoosastic & Elasticsearch version compatibility ?

### ISSUE THREAD

**Issue by mroberty8** [mongoosastic/mongoosastic#436]
Hi, I would like to know if current release of mongoosastic supported latest version of elasticsearch ?

I try to raise this error : `No handler for type [string] declared on field [myfield]`

It looks like a deprecation of `String` type in place of `Text` or `Keyword` type from elasticsearch since version 5.

Thanks in advance.

**Comment by larkin-nz** [comment]
Hey man,

It seems there is currently a pull request which contains a solution this, so hopefully, it will be patched soon.

See #420 

However, until this has been patched what you could do is the add the 'es_type' key to the field in your mongo schema where you have the 'type' set as a String, for example;

```js
  field: { 
    type: String,
    es_type: 'text',
    es_indexed:true 
  }
```

Hope this clears it up!

**Comment by idibidiart** [comment]
Has not been merged yet. Is this project in semi-neglected state at the moment? Or is their a legitimate reason why PR hasn't been merged?

**Comment by guumaster** [comment]
@idibidiart I'm a maintainer, but I'm not using Mongoose nor Elastic at the moment. Is hard for me to get free time to work on this. 

The reason for not merging it is because the tests are not passing for that PR. 

I'll publish a new version with updated deps when we get a branch with all green tests. 

Any help achieving that is welcome.

**Comment by guumaster** [comment]
We've released version 4.5.0 with support for Elastic version 6.x. Please update and check if its work.

**Comment by MostafaBekhet** [comment]
> We've released version 4.5.0 with support for Elastic version 6.x. Please update and check if its work.

does mongoosastic version 5.0.0 supports Elastic version 8.4.0 ?
